src/common/components/Navbar/ResponsiveMenu/index.js

```diff
@@ -1,53 +1,54 @@
 import React, { Component } from 'react';
 import PropTypes from 'prop-types';
 import Icon from 'antd/lib/icon';
-import Popover from 'antd/lib/popover';
 import Menu from 'antd/lib/menu';
+import Popover from 'antd/lib/popover';
```

Comment:
i am thinking shouldnt we import from common/components so that when we override the menu or popover or icon  we get the enhanced version?

Reply:
true